library/src/main/java/com/arlib/floatingsearchview/FloatingSearchView.java

```diff
@@ -419,34 +420,6 @@ protected void onLayout(boolean changed, int l, int t, int r, int b) {
         super.onLayout(changed, l, t, r, b);
 
         if (mIsInitialLayout) {
-
-            //we need to add 5dp to the mSuggestionsSection because we are
-            //going to move it up by 5dp in order to cover the search bar's
-            //shadow padding and rounded corners. We also need to add an additional 10dp to
-            //mSuggestionsSection in order to hide mSuggestionListContainer's
-            //rounded corners and shadow for both, top and bottom.
-            int addedHeight = 3 * Util.dpToPx(CARD_VIEW_CORNERS_AND_TOP_BOTTOM_SHADOW_HEIGHT);
```

Comment:
i haven't ran it. This code seems to be avoiding overlapping of content with the shadow that is drawn. Is it safe to remove this?

Reply:
It's meant for the "divider" between search query and the content - we don't use it since we set it as transparent, looks like the divider support aforementioned is better off getting removed altogether, let me test.

Reply:
Updated PR to remove divider customisation (different from suggestion list divider) along with usage of some hardcoded dp constants related to card shadow/padding adjustment in the FloatingSearchView class code